model/src/w3odatmd.F90

```diff
@@ -1,5 +1,20 @@
+!> @file w3odatmd.F90
+!> @ brief Define data structures to set up wave model grids and aliases
+!>         to use individual grids transparently.
+
 #include "w3macros.h"
 !/ ------------------------------------------------------------------- /
+!
+!> @author H. L. Tolman @date 22-Mar-2021
+!>
+!> @brief Define data structures to set up wave model grids and aliases
```

Comment:
brief is included twice? 

Also if the #include statement is moved to after all doxygen comments does it disappear from the doxygen documentation?

Reply:
oh or is one brief for the whole file and the other is for the module?

Reply:
Yes, to your second comment.  I did a brief for the file, and one for the module.  In these cases we might put the same, or similar, descriptions.  Since each is a separate component, it will be used for it's respective display.

Reply:
We discussed this offline, but for the rest of the community, unfortunately I could not find a way to get rid of the `#include w3macros.h` statement at the top.

Reply:
Thanks @MatthewMasarik-NOAA I"m going to resolve this.